Resources/Prototypes/Floof/Accents/word_replacements.yml

```diff
@@ -1,78 +1,84 @@
-    accent-german-words-25: accent-german-words-replace-25
-    accent-german-words-26: accent-german-words-replace-26
-    accent-german-words-27: accent-german-words-replace-27
-    accent-german-words-28: accent-german-words-replace-28
-    accent-german-words-29: accent-german-words-replace-29
-    accent-german-words-30: accent-german-words-replace-30
-    accent-german-words-31: accent-german-words-replace-31
-    accent-german-words-32: accent-german-words-replace-32
-    accent-german-words-32-2: accent-german-words-replace-32
-    accent-german-words-33: accent-german-words-replace-33
-    accent-german-words-34: accent-german-words-replace-34
-    accent-german-words-35: accent-german-words-replace-35
-    accent-german-words-36: accent-german-words-replace-36
-    accent-german-words-37: accent-german-words-replace-37
-    accent-german-words-38: accent-german-words-replace-38
-    accent-german-words-38-2: accent-german-words-replace-38
-    accent-german-words-39: accent-german-words-replace-39
-    accent-german-words-40: accent-german-words-replace-40
-    accent-german-words-41: accent-german-words-replace-41
-    accent-german-words-42: accent-german-words-replace-42
-    accent-german-words-43: accent-german-words-replace-43
-    accent-german-words-44: accent-german-words-replace-44
-    accent-german-words-45: accent-german-words-replace-45
-    accent-german-words-46: accent-german-words-replace-46
-    accent-german-words-47: accent-german-words-replace-47
-    accent-german-words-47-2: accent-german-words-replace-47
-    accent-german-words-48: accent-german-words-replace-48
-    accent-german-words-49: accent-german-words-replace-49
-    accent-german-words-50: accent-german-words-replace-50
-    accent-german-words-51: accent-german-words-replace-51
-    accent-german-words-52: accent-german-words-replace-52
-    accent-german-words-53: accent-german-words-replace-53
-    accent-german-words-53-2: accent-german-words-replace-53
-    accent-german-words-54: accent-german-words-replace-54
-    accent-german-words-55: accent-german-words-replace-55
-    accent-german-words-56: accent-german-words-replace-56
-    accent-german-words-56-2: accent-german-words-replace-56
-    accent-german-words-56-3: accent-german-words-replace-56
-    accent-german-words-57: accent-german-words-replace-57
-    accent-german-words-57-2: accent-german-words-replace-57
-    accent-german-words-58: accent-german-words-replace-58
-    accent-german-words-59: accent-german-words-replace-59
-    accent-german-words-60: accent-german-words-replace-60
-    accent-german-words-61: accent-german-words-replace-61
-    accent-german-words-61-2: accent-german-words-replace-61
-    accent-german-words-61-3: accent-german-words-replace-61
-    accent-german-words-61-4: accent-german-words-replace-61
-    accent-german-words-62: accent-german-words-replace-62
+    accent-french-words-1: accent-french-words-replace-1
```

Comment:
Why does this replace the german accent?

Reply:
o_O because i've got 3 braincells, it shouldn't be.  Lemme fix lmfao

Reply:
Yeah, I used german as the base to make it out of and forgot to make the word replacement separate.  What I get for getting in a hurry.